ros_packages/motors/pib_motors/pib_motors/update_bricklet_uids.py

```diff
@@ -47,13 +48,16 @@ def get_uids_from_db():
     json_data = json.loads(response.text)
     return [json_data['bricklets'][0]['uid'], json_data['bricklets'][1]['uid'], json_data['bricklets'][2]['uid']]
 
-def detect_uid_changes():
+def no_uids_in_database():
     """Check for changes between current databse and TinkerForge UIDs."""
     used_uids = get_uids_from_db()
+    count = 0
+    for uid_number, uid in enumerate(["AAA", "BBB", "CCC"]):
+        if uid == used_uids[uid_number]:
+            count += 1
```

Comment:
Instead of `["AAA", "BBB", "CCC"]` we should use the constants declared at the top of the file `[UID0, UID1, UID2]` or declare the list as another constant.

Also, why count all matching occurences instead of comparing the lists directly like so:
```python
return used_uids == ["AAA", "BBB", "CCC"]:
```